### PR TITLE
V4 generator bug

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

Generated files cause the simulator to crash if only the program methods change. 

## What was changed

- Fixed the bug related to generator files. 
- Added more error handling to initialization. 

## Intended Purpose

Improve user experience when running the simulation, specifically during the initialization. 

## Testing Completed

All unit tests pass: 
[results.txt](https://github.com/LDAR-Sim/LDAR_Sim/files/15168211/results.txt)

Manually ran simple case simulation with different generator files missing and by changing the method of a program.

### CODE REVIEW

File: LDAR_Sim/src/initialization/preseed.py [hunk 1]
```diff
@@ -52,29 +55,33 @@ def get_seed_timeseries(gen_dir) -> list[int]:
...
+    force_remake = False
     if not os.path.exists(gen_dir):
+        force_remake = True
         os.mkdir(gen_dir)
 
     if os.path.isfile(preseed_loc):
         preseed_val = get_emis_seed(gen_dir)
         if len(preseed_val) < n_sim:
-            print("Generating additional Random Seed values for emissions...")
+            force_remake = True
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/initialization/preseed.py:69]: Wondering if this should be true here - This is for the case where everything is the same, but now there are more simulations.

File: LDAR_Sim/src/programs/program.py [hunk 2]
```diff
@@ -73,8 +73,12 @@ def __init__(
         self._current_date: date = sim_start_date
         self.weather = weather
         self.daylight = daylight
-        self.duration_factor = prog_params[Program_Params.DURATION_ESTIMATE][Program_Params.DURATION_FACTOR],
-        self.duration_method = prog_params[Program_Params.DURATION_ESTIMATE][Program_Params.DURATION_METHOD],
+        self.duration_factor = prog_params[Program_Params.DURATION_ESTIMATE][
+            Program_Params.DURATION_FACTOR
```
Comment (ThomasGalesloot) [inline LDAR_Sim/src/programs/program.py:77]: LINTER?

**Review by ThomasGalesloot — APPROVED** [review]
See my comment before merging - Great work!

File: LDAR_Sim/src/initialization/preseed.py (hunk 1 above)
Comment (SEAJang) [reply LDAR_Sim/src/initialization/preseed.py]: You're correct. Removed.

**Pull request merged**